crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -108,6 +120,32 @@ mod tests {
             .fold(0, |sum, spend| sum + spend.create_coin.len());
         assert_eq!(create_coins, additions);
         assert_eq!(conditions.cost, cost);
+        // Generate a block with the same spend bundle and compare its cost
+        let program_spends: Vec<_> = bundle
+            .coin_spends
+            .iter()
+            .map(|coin_spend| {
+                (
+                    coin_spend.coin,
+                    &coin_spend.puzzle_reveal,
+                    &coin_spend.solution,
+                )
```

Comment:
rather than returning this tuple, you could just return the byte-size of serializing it. and then `sum()` that.

Reply:
right, and the only thing you do with the vector it returns is to call `.len()`. You could just compute the length instead

Reply:
this comment was not meant for the test, but the production code, here: https://github.com/Chia-Network/chia_rs/pull/680/files#diff-9ad3aa7496f5fdc7db0a33393d2c8bd18727a7924d3d98189433b273e3d15439R34-R42